Doc/library/code.rst

```diff
@@ -65,6 +65,22 @@ build applications which provide an interactive interpreter prompt.
    raises :exc:`OverflowError` or :exc:`ValueError` if the command contains an
    invalid literal.
 
+Overriding Console Output
+-------------------------
+
+All output is printed to :data:`sys.stderr` by default.  A user can override
+:meth:`InteractiveConsole.write` in a derived class to change that.
+Alternatively, it is possible to selectively redirect parts of the output:
+
+* The return values of successfully interpreted Python statements are printed
```

Comment:
This line is technically incorrect; statements don't have return values. I suppose it means the evaluated value of `ast.Expr` statements.